### PR TITLE
New vehicle data ClimateData

### PR DESCRIPTION
**Pull request by atiwari9**
This proposal is to add new vehicle data `ClimateData` to know cabin temperature, atmospheric pressure and external temperature.

### CODE REVIEW

**Comment by atiwari9** [comment]
@theresalech  - Ready for review

File: proposals/NNNN-New-vehicle-data-ClimateData.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+```
+
+## Potential downsides
+
+Author is not aware of any downsides to proposed solution. This proposal just enhances the SDL content.
+
+## Impact on existing code
+
+* SDL Core needs to be updated as per new API.
+* iOS/Java Suite needs to be updated to support getters/setters as per new API.
```
Comment (theresalech) [inline proposals/NNNN-New-vehicle-data-ClimateData.md]:
```suggestion
* iOS/Java Suite need to be updated to support getters/setters as per new API.
```

Comment (atiwari9) [reply proposals/NNNN-New-vehicle-data-ClimateData.md]: Done

**Comment by atiwari9** [comment]
@theresalech - I have update the proposal as per your review comments.

**Comment by atiwari9** [comment]
@theresalech - I have made additional updates to this proposal  to deprecate `externalTemperature` since `climateData` incorporates that now. 

This PR will not be pulled in for review in next meeting as there are other PRs in line before this one, so please review at your convenience.

**Pull request merged**